test/IISIntegration.FunctionalTests/Inprocess/SynchronousReadAndWriteTests.cs

```diff
@@ -23,7 +23,7 @@ public SynchronousReadAndWriteTests(IISTestSiteFixture fixture)
             _fixture = fixture;
         }
 
-        [ConditionalFact (Skip = "See https://github.com/aspnet/IISIntegration/issues/629")]
+        [ConditionalFact]
```

Comment:
`[Fact]` ?

Reply:
Everything in the repo is a conditional fact because we check if the update IIS Schema is installed or not: https://github.com/aspnet/IISIntegration/blob/dev/test/IISIntegration.FunctionalTests/Utilities/IISExpressSupportsInProcessHostingAttribute.cs